.travis.yml

```diff
@@ -1,6 +1,10 @@
 sudo: required
 dist: trusty
 
+branches:
+  only:
+    - "master"
```

Comment:
maybe we could remove 'master only' here? PR or other branches also need the check

Reply:
PR will always run, branches other than master should not run.

Reply:
OK